src/components/Contacts/ContactFilters/ContactFilters.test.tsx

```diff
@@ -47,6 +53,24 @@ describe('ContactFilters', () => {
         ContactFiltersDefaultMock.accountList.contactFilterGroups[1].name,
       ),
     ).toBeVisible();
+
+    userEvent.click(getByTestId('FilterListItemShowAll'));
+
+    expect(getByText('See More Filters')).toBeVisible();
+
+    await waitFor(() =>
+      expect(
+        getByText(
+          ContactFiltersDefaultMock.accountList.contactFilterGroups[0].name,
+        ),
+      ).not.toBeVisible(),
+    );
+
+    expect(
+      getByText(
+        ContactFiltersDefaultMock.accountList.contactFilterGroups[1].name,
+      ),
+    ).toBeVisible();
```

Comment:
I thought about making a separate test for the featured logic but just went with this. Let me know what you guys think.